getting-started/openshift/windows/installation.md

```diff
@@ -0,0 +1,265 @@
+2020/10/05 12:52:59 Added rule with port 22 to the security groups of your local IP
+2020/10/05 12:52:59 Added rule with port 3389 to the security groups of your local IP
+2020/10/05 12:52:59 Using existing Security Group: sg-06d1de22807d5dc48
+2020/10/05 12:57:30 External IP: 52.35.12.231
+2020/10/05 12:57:30 Internal IP: 10.0.90.193
+```
+
+#### Install {{site.prodnameWindows}}
+
+1. Prepare directory for Kubernetes files on Windows node.
+
+   ```powershell
+   mkdir c:\k
+   ```
+
+1. Copy the Kubernetes kubeconfig file from the master node (default, Location $HOME/.kube/config), to the file **c:\k\config**.
+
+1. Download the powershell script, **install-calico-windows.ps1**.
+
+   ```powershell
+   Invoke-WebRequest {{ "/scripts/install-calico-windows.ps1" | absolute_url }} -OutFile c:\install-calico-windows.ps1
+   ```
+1. Run the installation script, replacing the Kubernetes version with the version corresponding to your version of OpenShift:
+
+   ```powershell
+   c:\install-calico-windows.ps1 -KubeVersion <kube version> -ServiceCidr 172.30.0.0/16 -DNSServerIPs 172.30.0.10
+   ```
+
+1. Re-add the route for the AWS metdata endpoint. First, determine the interface
+   index for the `vEthernet (Ethernet 2)` adapter with `Get-NetAdapter`. For
+   example:
+
+   ```powershell
+   PS C:\> Get-NetAdapter
+
+   Name                      InterfaceDescription                    ifIndex Status       MacAddress             LinkSpeed
+   ----                      --------------------                    ------- ------       ----------             ---------
+   vEthernet (nat)           Hyper-V Virtual Ethernet Adapter             10 Up           00-15-5D-BA-0F-68        10 Gbps
+   Ethernet 2                Amazon Elastic Network Adapter                6 Up           0A-4F-1F-C4-E3-49        25 Gbps
+   vEthernet (Ethernet 2)    Hyper-V Virtual Ethernet Adapter #2          15 Up           0A-4F-1F-C4-E3-49        25 Gbps
+   ```
+
+   Then, add the route using the interface index:
+   ```powershell
+   PS C:\> New-NetRoute -DestinationPrefix 169.254.169.254/32 -InterfaceIndex <interface_index>
+   ```
+
+#### Configure kubelet
+
+Copy the previously downloaded file `wmcb.exe` and your worker's ignition file (worker.ign) to the Windows node.
+To download the worker.ign:
+
+```shell
+apiserver=$(oc get po -n  openshift-kube-apiserver -l apiserver=true --no-headers -o custom-columns=":metadata.name" | head -n 1)
+oc -n openshift-kube-apiserver exec ${apiserver} -- curl -ks https://localhost:22623/config/worker > worker.ign
+```
+
+Remote into the Windows node and configure the kubelet:
+
+```powershell
+ ./wmcb.exe initialize-kubelet --ignition-file worker.ign --kubelet-path c:\k\kubelet.exe
+```
+
+Next, configure kubelet to use Calico CNI:
+
+```powershell
+cp c:\k\config c:\k\kubeconfig
+./wmcb.exe configure-cni --cni-dir c:\k\cni --cni-config c:\k\cni\config\10-calico.conf
+```
+
+Restart the `calico-felix` service:
```

Comment:
Do you know why we need restart `calico-felix` service?

Reply:
This part I need to investigate a bit more. I've seen this happen every time so it might be a bug or something to do with what wmcb is doing.